### DISCUSSION TITLE
Welcome to open-web-calendar Discussions!

### DISCUSSION THREAD

**Discussion by niccokunzmann** [niccokunzmann/open-web-calendar#217] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DaveTavres** [comment]
Thank you so much for creating such a wonderful styling tool!

No matter what I do, I cannot get URLs / links in calendar items to open in a new tab... the links always open in the iframe.

It must be something with the Google Calendar item being created in the ical format.

Is there anything I can do to force links to open in a new tab?

Here are two calendars I've been trying:
https://calendar.google.com/calendar/ical/c_c195c26a0m1dpn4v5ot31pke4c%40group.calendar.google.com/public/basic.ics
https://calendar.google.com/calendar/ical/marcelinehistory%40gmail.com/public/basic.ics

**Reply by niccokunzmann** [reply]
At the moment, this requires editing the event in the calendar. But this can be made into an issue.

**Reply by DaveTavres** [reply]
Thank you... Could you point me in the right direction?

In the Google Calendar event, I don't know of a way to set the target of a link.

Or, did you mean something different?

**Reply by niccokunzmann** [reply]
> In the Google Calendar event, I don't know of a way to set the target of a link.

I meant exactly that. I do not know a way either. So, there is currently no known way of doing it until #287 is resolved.
If you find that you have some monetary support free, I think that 40$ should cover the costs - one can pledge them on the issue and still revoke payout if the feature is not implemented as expected.

**Reply by niccokunzmann** [reply]
@DaveTavres, this is fixed now and live from v1.32 on. Thanks for reporting!
